src/app/server/Server.cpp

```diff
@@ -481,4 +481,16 @@ CHIP_ERROR Server::SendUserDirectedCommissioningRequest(chip::Transport::PeerAdd
 }
 #endif // CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY_CLIENT
 
+void Server::ResumeSubscriptions()
+{
+#if CHIP_CONFIG_PERSIST_SUBSCRIPTIONS
+    CHIP_ERROR err = chip::app::InteractionModelEngine::GetInstance()->ResumeSubscriptions();
```

Comment:
Is the InteractionModelEngine pointer not in `Server` class?

Reply:
I don't see one. There are two references to it and both use `chip::app::InteractionModelEngine::GetInstance()`.